app/controllers/v0/facilities/ccp_controller.rb

```diff
@@ -55,16 +55,20 @@ def api
     @api ||= Facilities::PPMS::Client.new
   end
 
+  def per_page
+    Integer(search_params[:per_page] || BaseFacility.per_page)
+  end
+
   def page
-    Integer(params[:page] || 1)
+    Integer(search_params[:page] || 1)
   end
 
   def pages(ppms_results)
     total = ppms_results.length
     {
       current_page: page,
-      per_page: BaseFacility.per_page,
-      total_pages: Integer(ppms_results.length / BaseFacility.per_page + 1),
+      per_page: per_page,
+      total_pages: page + 1,
```

Comment:
I'm confused here.  `total_pages` is always `current_page + 1`?
So, if we're on page 1, `total_pages` is 2 even if there are 7 pages worth of results...?

Reply:
The PPMS response does not include any pagination metadata. This is a hack to allow the front end to request the next page of results.